main.js

```diff
@@ -105,22 +114,43 @@
     playerOneDisplay.innerText = challengerNames[0].value;
     playerTwoDisplay.innerText = challengerNames[1].value;
     disableInput()
+    guessErrorMessage()
     checkGuesses()
-    guessOneDisplay.innerText = challengerGuesses[0].value;
-    guessTwoDisplay.innerText = challengerGuesses[1].value;
     clearFields()
     disableButtons()
     gameResetButton.removeAttribute('disabled')
   }
 
+  function guessErrorMessage() {
+    console.log(maxNumber)
+    if (parseInt(challengerGuesses[0].value) > maxNumber ||
+        parseInt(challengerGuesses[0].value) < minNumber) {
+          guessOneDisplay.innerText = 'N/A';
+          if (numberOfGuesses !== 0) {
```

Comment:
We should make a guess even if it is not within range it still counts as a guess.

Reply:
I think we should just disable the submit button if it's outside the range so it doesn't count as a guess.